demos/HST/S6_wfc3_template.ecf

```diff
@@ -5,7 +5,7 @@
 allapers        False # Run S6 on all of the apertures considered in S5? Otherwise will use newest output in the inputdir
 
 # Plotting parameters
-y_params        ['rp^2', 'u1', 'u2', 'c0', 'c1', 'scatter_mult']  # The parameter name as entered in your EPF file in Stage 5. rp^2, 1/r1, 1/r4, fn (nightside flux), pc_amp (phase curve amplitude), and pc_offset (phase curve offset), pc_amp2 (second order phase curve amplitude), and pc_offset2 (second order phase curve offset) are also permitted.
+y_params        ['rp^2', 'u1', 'u2', 'c0', 'c1', 'scatter_mult']  # The parameter name as entered in your EPF file in Stage 5. rp^2, 1/r1, 1/r4, fn (nightside flux), pc_amp (phase curve amplitude), and pc_offset (phase curve offset), amp_order# (first/second order phase curve amplitude), and offset_order# (first/second order phase curve offset) are also permitted.
```

Comment:
I worry that we're going to have folks writing issues saying they don't get the expected outputs when they use `amp_order#` since they won't figure out that you mean to do something like `amp_order1`...

Reply:
Sigh... I could totally see that happening.